res/controllers/Numark-Mixtrack-3.midi.xml

```diff
@@ -0,0 +1,918 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!-- Numark Mixtrack (Pro) 3 MIDI preset v1.3 -->
+<MixxxControllerPreset mixxxVersion="1.12.0+" schemaVersion="1">
+    <info>
+	<name>Numark Mixtrack (Pro) 3</name>
+	<author>Stéphane Morin</author>
+	<description>The Numark Mixtrack 3 and Numark Mixtrack Pro 3 are the same controller except that the Pro version has an integrated sound card.</description>
+        <wiki>http://www.mixxx.org/wiki/doku.php/numark_mixtrack_pro_3</wiki>
```

Comment:
Is there a forum post for this mapping yet? Would you like to make a placeholder forum post and put it here? 
The syntax is:

```<forums>url</forums>``` 
make sure to escape the ```&``` as ```&amp;```